### ISSUE TITLE
Accessibility is lacking

### ISSUE THREAD

**Issue by zcbenz** [chieapp/chie#7]
Support for accessibility is basically unusable due to lots of custom drawn interface, we need to add APIs in libyue to provide accessibility hints and then implement accessibility in Chie.
